internal/lm/mig-strategy_test.go

```diff
@@ -83,7 +83,7 @@ func TestMigStrategyNoneLabels(t *testing.T) {
 				rt.NewFullGPU(),
 				rt.NewFullGPU(),
 			},
-			timeSlicing: spec.ReplicatedResources{
+			timeSlicing: &spec.ReplicatedResources{
```

Comment:
We need to change the type in the struct definition above too.

Reply:
I did it a bit differently - I kept the struct definition as-is, and changed the way the test cases are built.

Reply:
Sure. Let me re-review.

Reply:
Blah, there's still CI problems - will make sure it passes before I push again.